modules/programs/zellij.nix

```diff
@@ -1,32 +1,29 @@
 { config, lib, pkgs, ... }:
 
-with lib;
-
 let
+  inherit (lib) mkIf mkOption types;
 
   cfg = config.programs.zellij;
   yamlFormat = pkgs.formats.yaml { };
-  zellijCmd = getExe cfg.package;
-
 in {
-  meta.maintainers = [ hm.maintainers.mainrs ];
+  meta.maintainers = [ lib.maintainers.khaneliman lib.hm.maintainers.mainrs ];
 
   options.programs.zellij = {
-    enable = mkEnableOption "zellij";
+    enable = lib.mkEnableOption "Zellij";
 
     package = mkOption {
```

Comment:
Drive-by comment: could be `mkPackageOption "Zellij" { default = "zellij"; }`

Reply:
Good catch, didn't notice it!